baselines/diabetic_retinopathy_detection/experiments/config/imagenet21k_vit_base16_finetune.py

```diff
@@ -87,20 +96,81 @@ def get_config():
   # This is "no head" fine-tuning, which we use by default
   config.model.representation_size = None
 
-  # Optimizer section
+  # * Preprocessing. *
+
+  # Input resolution of each retina image. (Default: 512)
+  config.pp_input_res = 512  # pylint: disable=invalid-name
+  pp_common = f'|onehot({config.num_classes})'
+  config.pp_train = (
+      f'diabetic_retinopathy_preprocess({config.pp_input_res})' + pp_common)
+  config.pp_eval = (
+      f'diabetic_retinopathy_preprocess({config.pp_input_res})' + pp_common)
+
+  # * Training Misc. *
+  config.batch_size = 512  # using TPUv3-64
+  config.seed = 0  # Random seed.
+  config.shuffle_buffer_size = 15_000  # Per host, so small-ish is ok.
+
+  # * Optimization. *
   config.optim_name = 'Momentum'
   config.optim = ml_collections.ConfigDict()
   config.loss = 'softmax_xent'  # or 'sigmoid_xent'
-  # config.loss = 'sigmoid_xent'
-
-  # config.lr = ml_collections.ConfigDict()
-
-  # Set as command line arguments now, for wandb compatibility.
-  # config.grad_clip_norm = 20.0
-  # config.weight_decay = None  # No explicit weight decay
-  # config.lr.base = 0.003
-  # config.lr.warmup_steps = 500
-  # config.lr.decay_type = 'cosine'
+  config.lr = ml_collections.ConfigDict()
+  config.grad_clip_norm = 1.0  # Gradient clipping threshold.
+  config.weight_decay = None  # No explicit weight decay.
+  config.lr.base = 0.003
+  config.lr.warmup_steps = 500
+  config.lr.decay_type = 'linear'
+
+  # The dataset is imbalanced (e.g., in Country Shift, we have 19.6%, 18.8%,
+  # 19.2% positive examples in train, val, test respectively).
+  # None (default) will not perform any loss reweighting.
+  # 'constant' will use the train proportions to reweight the binary cross
+  #   entropy loss.
+  # 'minibatch' will use the proportions of each minibatch to reweight the loss.
+  config.class_reweight_mode = None
+
+  # * Evaluation Misc. *
+  config.only_eval = False  # Disables training, only evaluates the model.
+  config.use_validation = True  # Whether to use a validation split.
+  config.use_test = True  # Whether to use a test split.
+
+  # * Step Counts. *
+  config.total_steps = 10_000
+  config.log_training_steps = 100
+  config.log_eval_steps = 1000
+  # NOTE: eval is very fast O(seconds) so it's fine to run it often.
+  config.checkpoint_steps = 1000
+  config.checkpoint_timeout = 1
 
   config.args = {}
   return config
+
+def get_sweep(hyper):
```

Comment:
Since you're not using `get_sweep`, let's remove this for now and I'll add it as part of my PR (https://github.com/google/uncertainty-baselines/pull/693). Namely, let's merge this PR, then I'll resolve the conflicts for my branch, and try to get mine submitted.